shex/patientConsentShape.shex

```diff
@@ -0,0 +1,51 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v030/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+:identifierShape IRI {
+    a [sio:SIO_000115] ;
+    sio:SIO_000020 @:consentRoleShape ;
+    sio:SIO_000300 xsd:string
+}
+
+:personShape IRI { 
+    a [sio:SIO_000498] ;
+    sio:SIO_000228 @:consentRoleShape ;
+    sio:SIO:000008 @:consentAttributeShape
+}
+
+:consentRoleShape IRI {
+    a [obo:OBI_0000093] ;
+    a [sio:SIO_000016] ;
+    rdfs:label xsd:string;
+    sio:SIO_000356 @:consentProcessShape
+}
+
+:consentProcessShape IRI {
+    a [obo:OBI_0000810] ;
+    sio:SIO_000669 xsd:date ;
+    sio:SIO_000229 @:patientconsentOutputShape ;
+    sio:SIO_000230 @:consentOutputShape
+}
+
+:patientconsentOutputShape IRI {
+    a [sio:SIO_000015] ;
+    rdfs:label xsd:string ;
+    sio:SIO_000669 xsd:date ;
+    sio:SIO_000300 xsd:string ;
```

Comment:
Should it be string or IRI? In the Turtle, it is the string format of an IRI. Is it on purpose? @rajaram5

Reply:
Mark establishes `sio:has-value` as string in output node and attribute's type node as IRI, but in could be discussed becuase it comes from YARRRML template's structure: https://github.com/ejp-rd-vp/CDE-semantic-model-implementations/blob/master/YARRRML_Transform_Templates/templates/patient_consent_yarrrml_template.yaml#L146

Reply:
Oh you are right!  `sio:has-value` is the datatype property and can only be followed by a literal in a triple.